Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,46 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you care about the environment?
+default persistent._mas_pm_cares_environment = None
+
+init 5 python:
+    addEvent(
+        Event(persistent.event_database,eventlabel="monika_environment",category=['nature', 'life'],prompt="The environment",random=True))
+ 
+label monika_environment:
+    m 2eud "Hey [player], I've been wondering..."
+    $ _history_list.pop()
+    menu:
+        m "Do you care about the environment{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_cares_environment = True
+            m 3sub "That's wonderful!"
+            m 3hsb "I'm so glad to hear that, player."
+        
+        "No.":
+            $ persistent._mas_pm_cares_environment = False
+            m 1rkb "That's alright, you're entitled to your opinion."
+            m 3rkb "Not everyone is a nature person, after all."
```

Comment:
For the `No.` path, I think you could go in depth about why the player might think that way instead of dismissing it and continuing, This is one reason I thought about potentially having this pm var topic as its own topic, so both topics can focus entirely on their own topic. A dedicated topic would allow for much more depth in the options, and keeping a separate topic for Monika's thoughts would allow the topic to remain randomed, which is ideal, especially considering how little of the topic here actually covers the player's thoughts.

Reply:
> I'm wondering if it would be better to separate the topics, so they can each focus in depth on their own dedicated area, instead of mixing in a small part of something else.

Yeah, I think that's a good idea. I'm not happy with how the dialogue flows ATM because I really wanted to have the ending change depending on your choice (see the failed if/else commit) so I'll rewrite it into two different sets of dialogue entirely for Yes and No respectively so that it's more like the mint ice cream topic since that one seems to flow pretty well. I'm glad the code works, at least.

> A dedicated topic would allow for much more depth in the options, and keeping a separate topic for Monika's thoughts would allow the topic to remain randomed, which is ideal, especially considering how little of the topic here actually covers the player's thoughts.

I've been back and forth on this in my mind because I'm not sure how neutral Monika would actually be able to stay in a situation where the player says No, but considering that she's quite knowledgeable in most of her lines I'm probably just underestimating her. I'll reformat things.

Reply:
Keeping in mind that Monika was in debate club, and it's key to phrase things neutrally, even if you disagree, and to keep an open mind. She doesn't have to be neutral at her core, but I think she should have an open mind and be willing to try and see the other perspective.

Reply:
Yeah, I'll see what I can do. The "No" branch will definitely be harder than the "Yes" branch, but I'll figure something out.